uint/benches/bigint.rs

```diff
@@ -16,9 +16,23 @@
 
 extern crate core;
 extern crate test;
+#[macro_use]
 extern crate uint;
 
-use uint::{U256, U512};
+construct_uint! {
+	pub struct U256(4);
+}
+
+construct_uint! {
+	pub struct U512(8);
+}
+
+impl U256 {
+	#[inline(always)]
+	pub fn full_mul(self, other: U256) -> U512 {
```

Comment:
It's possible to avoid this duplication once we have `primitive-types`. For now, I would rather keep it like this to avoid "circular PR".

Reply:
good idea!

Reply:
@sorpaas can you please add an issue for this so we don't forget? "Once PR #xx is merged, remove|deprecate "full_mul"… etc"?

Reply:
Added as: https://github.com/paritytech/parity-common/issues/91